PlagiarismSubmissionSubmitListener.php

```diff
@@ -3,8 +3,8 @@
 /**
  * @file plugins/generic/plagiarism/PlagiarismSubmissionSubmitListener.php

```

Comment:
Looking at the `diff`, this file appears to have DOS line endings; please convert over to UNIX-style.